crates/vision/src/image_segmenter.rs

```diff
@@ -743,27 +726,37 @@ mod tests {
 
     #[test]
     fn image_with_three_vertical_increasing_segments_with_median() {
+        let row = |y| {
+            [
+                YCbCr422::new(y, 0, 0, 0),
+                YCbCr422::new(y, 0, 0, 0),
+                YCbCr422::new(y, 0, 0, 0),
+            ]
+        };
         let image = YCbCr422Image::from_ycbcr_buffer(
-            1,
+            3,
```

Comment:
And this one?

Reply:
Same thing here, you can't take a horizontal median of three on a 1 pixel wide image.